resources/assets/components/FilterBar/index.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+
+import DropdownFilter from '../DropdownFilter';
+import MultiValueFilter from '../MultiValueFilter';
+
+class FilterBar extends React.Component {
+  constructor() {
+    super();
+
+    this.state = {
+      status: 'accepted',
```

Comment:
This probably doesn't need a default value since on component mount, it doesn't do any work to determine what we are filtering by. It's only on submit of this component, that it should do the work of updating the state of the parent component.